practical/swarm_demo/install.sh

```diff
@@ -25,13 +34,16 @@ function provision_machines {
     docker $(docker-machine config keystore) run -d -p "8500:8500" -h "consul" progrium/consul -server -bootstrap
 
     echo "Creating Swarm master..."
-    docker-machine create -d virtualbox --swarm --swarm-master --swarm-discovery="consul://$(docker-machine ip keystore):8500" --engine-opt="cluster-store=consul://$(docker-machine ip keystore):8500" --engine-opt="cluster-advertise=eth1:2376" swarm-master
+    docker-machine create -d virtualbox --swarm --swarm-master --swarm-discovery="consul://$(docker-machine ip keystore):8500" --engine-opt="cluster-store=consul://$(docker-machine ip keystore):8500" --engine-opt="cluster-advertise=eth1:2376" swarm-master && sleep 2
+    docker-machine start swarm-master && sleep 20
```

Comment:
Put 20 after swarm creation because a few times the node-01 didn't get properly included in the cluster. Not necessary in most cases to be honest.